docs/migration/00b_Electronic_structure_with_v0.5.ipynb

```diff
@@ -0,0 +1,729 @@
+    "The contents of the `drivers` submodule has been split into multiple places. The table below summarizes where each of the components of `qiskit_nature.drivers.second_quantization` has ended up.\n",
+    "\n",
+    "| Legacy component | New location |\n",
+    "|:-----------------|:-------------|\n",
+    "| BaseDriver | `qiskit_nature.second_q.drivers.BaseDriver` |\n",
+    "| BasisType | **removed** |\n",
+    "| ElectronicStructureDriver | `qiskit_nature.second_q.drivers.ElectronicStructureDriver` |\n",
+    "| ElectronicStructureDriverType | **removed** |\n",
+    "| ElectronicStructureMoleculeDriver | **removed** |\n",
+    "| FCIDumpDriver | `qiskit_nature.second_q.formats.fcidump.FCIDump` |\n",
+    "| GaussianDriver | `qiskit_nature.second_q.drivers.GaussianDriver` |\n",
+    "| HDF5Driver | **removed** |\n",
+    "| InitialGuess | `qiskit_nature.second_q.drivers.InitialGuess` |\n",
+    "| MethodType | `qiskit_nature.second_q.drivers.MethodType` |\n",
+    "| PSI4Driver | `qiskit_nature.second_q.drivers.Psi4Driver` |\n",
+    "| PyQuanteDriver | **removed** |\n",
+    "| PySCFDriver | `qiskit_nature.second_q.drivers.PySCFDriver` |\n",
+    "\n",
+    "Furthermore, the two components from `qiskit_nature.drivers` were moved like so:\n",
+    "\n",
+    "| Legacy component | New location |\n",
+    "|:-----------------|:-------------|\n",
+    "| Molecule | `qiskit_nature.second_q.formats.molecule_info.MoleculeInfo` |\n",
+    "| UnitsType | `qiskit_nature.units.DistanceUnit` |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "549ee735",
+   "metadata": {},
+   "source": [
+    "A few notes are worth adding:\n",
+    "\n",
+    "- The driver for [pyquante2](https://github.com/rpmuller/pyquante2) was removed, because its source code has not been updated for more than 3 years. Originally, it was supported to enable usage of Qiskit Nature on Windows, however, since then [psi4](https://github.com/psi4/psi4) has also added Windows support and the integration of the [Windows Subsystem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) has opened up further development possibilities.\n",
+    "- The `HDF5Driver` was removed in favor of supporting the standardized [QCSchema](https://github.com/MolSSI/QCSchema) (see `qiskit_nature.second_q.formats.qcschema`).\n",
+    "- The `ElectronicStructureMoleculeDriver` was removed because we are steering towards a future with tighter, plugin-like integration with classical codes, making the concept of drivers where Qiskit starts a classical simulation outdated. You can still use the `.from_molecule(...)` methods of the remaining drivers in combination with the `MoleculeInfo` class.\n",
+    "- The `MoleculeInfo` has become a pure data container and no longer supports _degrees of freedom_."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64cfb910",
+   "metadata": {},
+   "source": [
+    "### Using drivers that were migrated to `qiskit_nature.second_q.drivers`"
```

Comment:
FYI These were not run to see the results here (of course CI will but its not here in this I am looking at to review thats all)